integrate-existing-app/backend/.env.local

```diff
@@ -0,0 +1,8 @@
+PORT=8000
+# The JWT Secret to sign the JWT token
+JWT_SECRET=PUT_YOUR_SECRET_HERE
+# API Base URL (copy this from CARTO Workspace -> Developers)
+CARTO_API_BASE_URL=https://gcp-us-east1.api.carto.com
+# ClientID and Secret of your Machine to Machine Application
+CARTO_CLIENT_ID=GlLAKY2XJUAvXfx5XA1TdGoAtOCooyeL
+CARTO_CLIENT_SECRET=Hf6N3fucJDUjcX6lYaqUx4bQwr-gp7ch6U3g11bbdwcv0m6gul1Yr8Toa0OU9sb9
```

Comment:
@alasarr oops, not intended, right?

Reply:
`integrate-existing-app/backend/.gitignore` was not excluding the .local I have fixed it. I have also removed those apps as we're in a public repo